### PR TITLE
Format kodi-dev-kit

### PR DESCRIPTION
**Pull request by garbear**
## Description

This PR applies clang-format to the kodi-dev-kit. It also formats a stray input file.

Formatted with the command:

```bash
clang-format-11 -style=file -i \
  $(find \
    xbmc/addons/kodi-dev-kit \
    xbmc/cores/RetroPlayer \
    xbmc/games \
    xbmc/input \
    xbmc/peripherals \
    xbmc/smarthome \
    xbmc/ServiceBroker.* \
    xbmc/ServiceManager.* \
    -name "*.cpp" -o -name "*.h" \
  )
```

## Motivation and context

About a year ago I ran clang-format in my dev branch so that I could more easily integrate new Game API features. I figured it's time to upstream.

## How has this been tested?

Tested in all RetroPlayer builds I've done for about a year now.

## What is the effect on users?

* None

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

### CODE REVIEW

**Review by howie-f — APPROVED** [review]

File: xbmc/input/actions/ActionIDs.h [hunk 1]
```diff
@@ -381,18 +382,23 @@ constexpr const int ACTION_TOGGLE_DIGITAL_ANALOG = 202; //!< switch digital <->
...
-constexpr const int ACTION_TELETEXT_YELLOW = 217; //!< Teletext Color button <b>Yellow</b> to control TopText
-constexpr const int ACTION_TELETEXT_BLUE = 218; //!< Teletext Color button <b>Blue</b> to control TopText
+constexpr const int ACTION_TELETEXT_RED =
+    215; //!< Teletext Color button <b>Red</b> to control TopText
+constexpr const int ACTION_TELETEXT_GREEN =
+    216; //!< Teletext Color button <b>Green</b> to control TopText
+constexpr const int ACTION_TELETEXT_YELLOW =
+    217; //!< Teletext Color button <b>Yellow</b> to control TopText
+constexpr const int ACTION_TELETEXT_BLUE =
+    218; //!< Teletext Color button <b>Blue</b> to control TopText
```
Comment (fuzzard) [inline xbmc/input/actions/ActionIDs.h:392]: Personally im not a fan of the changes in ActionIDs.h
I think it makes things less readable, i think we would be better off doing clang format off/on for this block.

But whatever people want is fine, just thought id voice my opinion.

**Review by fuzzard — APPROVED** [review]
Everything else looks fine. Not a fan of the formatting for ActionIDs.h, but whatever people want to go with is fine.

File: xbmc/input/actions/ActionIDs.h (hunk 1 above)
Comment (garbear) [reply xbmc/input/actions/ActionIDs.h:392]: Easy solution, we comment like a normal person would:

```c++
//!< Teletext Color button <b>Red</b> to control TopText
constexpr const int ACTION_TELETEXT_RED = 215;

//!< Teletext Color button <b>Green</b> to control TopText
constexpr const int ACTION_TELETEXT_GREEN = 216;

//!< Teletext Color button <b>Yellow</b> to control TopText
constexpr const int ACTION_TELETEXT_YELLOW = 217;

//!< Teletext Color button <b>Blue</b> to control TopText
constexpr const int ACTION_TELETEXT_BLUE = 218;
```

Thanks for taking a look. I didn't put as much work into this PR as the mongo savestate manager one. I'll update the code.

Comment (fuzzard) [reply xbmc/input/actions/ActionIDs.h:392]: also very acceptable, just looked like more work, haha

Comment (garbear) [reply xbmc/input/actions/ActionIDs.h]: clang made pretty bad work out of the pvr inclues too. I went through the entire PR by hand, should be good now.

**Comment by garbear** [comment]
@ksooo Are you find with the changes to the pvr includes?

File: xbmc/input/actions/ActionIDs.h (hunk 1 above)
Comment (garbear) [reply xbmc/input/actions/ActionIDs.h]: We change 180 lines now down from 460

**Review by ksooo — APPROVED** [review]
PVR changes look good to me. 👍

**Pull request merged**